docs/docs/databases/connect-db/index.mdx

```diff
@@ -31,7 +31,7 @@ engine instance via the console / metadata APIs / CLI:
 
 Head to `Data -> Manage -> Connect database`
 
-<Thumbnail src='/img/databases/connect-default-db.png' alt='Connect default database' width='1000px' />
+<Thumbnail src='/img/databases/connect-db_step-1_console_2.10.1.png' alt='Connect default database' width='1000px' />
```

Comment:
Please leave the setting on this image on "Environment Variable" as that is the preferred way of connecting to DBs. 

Please crop like the existing image as per the original and not the whole page. EG:
<img width="1042" alt="Screenshot 2022-10-18 at 14 21 55" src="https://user-images.githubusercontent.com/94021366/196441852-19022569-9d0d-42a3-85f7-78ae8cda113c.png">